Source/Rio.Web/src/app/pages/water-trading-scenario/water-trading-scenario.component.ts

```diff
@@ -108,14 +87,20 @@ export class WaterTradingScenarioComponent implements OnInit, AfterViewInit {
             ],
             timeDimension: true,
             timeDimensionOptions: {
-                timeInterval: this.getISOString("January 2020") + "/" + this.getISOString("December 2020"),
+                timeInterval: this.getISOString(this.availableScenarioInfo[0].RunDate) + "/" + this.getISOString(this.availableScenarioInfo[this.availableScenarioInfo.length-1].RunDate), //this.getISOString(JSON.parse(this.availableScenarioInfo[0].FileDetails).RunResultName) + "/" + this.getISOString(JSON.parse(this.availableScenarioInfo[this.availableScenarioInfo.length-1].FileDetails).RunResultName),
```

Comment:
maybe extract variables here so we don't have repeating "this.availableScenarioInfo[...] calls